shared/js/ui/pages/options.es6.js

```diff
@@ -34,6 +35,12 @@ Options.prototype = window.$.extend({},
             window.$('.js-report-site-link')
                 .click(this._onReportSiteClick.bind(this))
 
+            $('#options-sub-header')[0].innerHTML = t('options:optionsSubHeader.title')
```

Comment:
this is using jQuery to find elements, then immediately de-referencing the raw element via the index access.

Instead, you can just do 

```js
$('#options-sub-header').text(t('options:optionsSubHeader.title')) 
```

or 

```js
$('#options-sub-header').html(t('options:optionsSubHeader.title')) 
```

Reply:
Agreed, much cleaner, thanks!